Makefile

```diff
@@ -25,29 +25,20 @@ disperse-test-blob:
 clean:
 	rm bin/eigenda-proxy
 
+# Runs all tests, excluding e2e and fuzz tests
 test-unit:
-	go test ./... -parallel 4
+	go test `go list ./... | grep -v ./e2e | grep -v ./fuzz` -parallel 4
 
-# Local V1/V2 E2E tests, leveraging op-e2e framework. Also tests the standard client against the proxy.
-test-e2e-local:
-	INTEGRATION=true go test -timeout 1m ./e2e -parallel 4
-	INTEGRATION_V2=true go test -timeout 1m ./e2e -parallel 4
-
-# E2E tests against holesky testnet
-# Holesky is currently broken after recent pectra hardfork.
-# This test is thus flaky depending on whether the testnet producing blocks or not
-# at the time it is run...
-# In good cases it runs in ~20 mins, so we set a timeout of 30 mins.
-# The test failing in CI is currently expected however, so expect to have to re-run it.
-# See https://dora.holesky.ethpandaops.io/epochs for block production status.
-test-e2e-holesky:
+# E2E tests, leveraging op-e2e framework. Also tests the standard client against the proxy.
+# If holesky tests are failing, consider checking https://dora.holesky.ethpandaops.io/epochs for block production status.
+test-e2e:
 	# Add the -v flag to be able to observe logs as the run is happening on CI
 	# given that this test takes >20 mins to run. Good to have early feedback when needed.
-	TESTNET=true go test -v -timeout 30m ./e2e  -parallel 4
+	go test -v -timeout 30m ./e2e -parallel 4
```

Comment:
think you said we can reduce the timeout to 10 mins now?

Reply:
or at least update the comment saying that these tests run in ~5 mins

Reply:
[3435b21](https://github.com/Layr-Labs/eigenda-proxy/pull/337/commits/3435b21e63d380abac50cbf61c51f58f1e54e117)